make/tasks/homebrew/release-formulas.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python3
+
+import os, re, logging, subprocess
+from github import Github, Auth
+from sys import platform
+
+# Env vars
+target_repo_name = os.environ.get("TARGET_REPO_NAME", None)
+if target_repo_name is None:
+    raise Exception("No target repo name, please set `TARGET_REPO_NAME` env")
+
+access_token = os.environ.get("GITHUB_ACCESS_TOKEN", None)
+if access_token is None:
+    raise Exception("No access token, cannot authenticate with GitHub please set `GITHUB_ACCESS_TOKEN` env")
+
+branch_prefix = os.environ.get("RELEASE_BRANCH_PREFIX", 'release')
+git_email = os.environ.get("TUIST_GIT_EMAIL", "")
+git_user = os.environ.get("GITHUB_REPOSITORY_OWNER", "")
+
+# Constants
+target_repo_url = f'https://github.com/{target_repo_name}' # Supporting github for now
+tuist_package_repo_url = "https://github.com/tuist/tuist"
+
+# MARK: - Helpers
+
+def _set_logging():
+    LOG_LEVEL = os.environ.get("LOG_LEVEL", "INFO")
+    print("LOG_LEVEL = {}".format(LOG_LEVEL))
+
```

Comment:
hat advantages does using the `.format(...)` method offer in comparison to `f"..."` besides the fact that the code appears more verbose?

```suggestion
    print(f"LOG_LEVEL = {LOG_LEVEL}")
```

Reply:
I have a concern about the fact that the script is written in `Python`, as the Homebrew Formula will be written in `Ruby` or `Bash`. Do you think it would be better to migrate the script to Ruby if it gets accepted?

I mention this to maintain a somewhat more controlled and less open ecosystem in terms of different programming languages.

Reply:
I chose Python because that is the scripting language I feel comfortable with and would take me the least amount of time. I couldn't find any best practices around what is preferred in the Tuist ecosystem.

Looking in the repo I see most scripts are in bash. As basically 90% of the actions I'm running are in bash, I can  convert it to bash. (or even Swift if that is preferred)

Before I put in the effort, I'd love to learn first if this workflow is acceptable by the maintainers.